apps/patient/src/views/ReadyBy.tsx

```diff
@@ -85,17 +72,10 @@ export const ReadyBy = () => {
 
     setOrder({
       ...order,
-      readyBy,
-      readyByTime
```

Comment:
Why is readyByTime being removed here but used in the datadog action?

Reply:
Oh! Good catch. Was thinking i don't use it later, but obviously still need to save it lol.

Reply:
added back